### ISSUE TITLE
A11y_SQL Azure Data Studio_Import ExtensionModify Columns_Keyboard: The focus after using escape key on the list of items in column 2 row 2 drop down button is seen on the edit field beside it.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#12980]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)– Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Oct2020;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG2.4.3;#Benchmark;#Keyboard;#AINotInScope;

**Environment Details:**
Application Name: SQL Azure Data Studio
Version 1.22.1 User Setup
Windows Version: Windows10

**Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Tab to server drop down and tab again to button new connection press enter key and add the server details.
3. Tab to the connected server and press enter key, use down key and select any database keep_MDW Right click on/ open the context menu for a database and select Data-tier application wizard from the context menu
4. Add data in specify input file screen and tab to "next" button and press enter.
5. Press enter on next button.
6. Tab to drop down button in cell column 2 row2 and press enter key.
7. Once the items have been expanded use the escape key to close the list.

**Actual:**
On using the escape key to close the list for the drop-down button in column 2 row2 the focus in on the edit field in column 2 row2 .

**Expected:**
The focus on using escape key on the list should be on the drop down button that opened the list.

**User Impact:**
The keyboard users will have to use an extra tab to move to the next column and this would disrupt the focus order.

**Attachment:**

[#12980_A11y_SQL Azure Data Studio_Import ExtensionModify Columns_Keyboard.zip](https://github.com/microsoft/azuredatastudio/files/5401664/12980_A11y_SQL.Azure.Data.Studio_Import.ExtensionModify.Columns_Keyboard.zip)

**Comment by fsteffi** [comment]
hi @alanrenmsft has the item "Data-tier application wizard " been removed

GithubTags:#Regressed:02-20-23;

**Comment by fsteffi** [comment]
githubtags:#closed;

Version: 1.42.0-insider
Commit: 872095ed849537d6c96373eedcd187b3f707c460
Date: 2023-02-23T06:29:37.626Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.25300

[12980.webm](https://user-images.githubusercontent.com/24580570/220862574-652805e8-3402-4c69-8dea-05541fd25bd7.webm)
